jerry-core/ecma/builtin-objects/ecma-builtin-date-prototype.cpp

```diff
@@ -38,6 +42,55 @@
  */
 
 /**
+ * Insert leading zeros to a string of a number if needed.
+ */
+static void
+ecma_date_insert_leading_zeros (ecma_string_t **str_p, /**< input/output string */
```

Comment:
I'm asking because neighbour function is named with `ecma_builtin_date_` : https://github.com/Samsung/jerryscript/pull/336/files#diff-4c3f6b953c0f51c11d2a079e9dcc10f8R103